components/sections/Projects/index.jsx

```diff
@@ -0,0 +1,58 @@
+import projects from "/data/projects";
+import Image from "next/image";
+import Title from "../../ui/Title";
+import Wrapper from "../../Wrapper";
+import ProjectItem from "./ProjectItem";
+import { useState } from "react";
+
+const Projects = () => {
+  const [active, setActive] = useState();
```

Comment:
can you set the default value so that it can be clear to know which kind of data this state is storing